great_expectations/datasource/fluent/pandas_datasource.py

```diff
@@ -114,13 +114,8 @@ def _get_reader_method(self) -> str:
     def test_connection(self) -> None:
         ...
 
-    @property
-    @override
-    def batch_request_options(self) -> tuple[str, ...]:
-        return tuple()
-
     @override
-    def get_batch_request_options_keys(self, partitioner):
+    def get_batch_request_options_keys(self, partitioner: Optional[Partitioner] = None):
```

Comment:
Can we annotate the return?

Reply:
I was under the impression that we got this typing from the parent class, so adding types here is redundant. But if i'm wrong about that, I'm happy to update!

Reply:
i gave this some more thought, and i think you're right, even though the type is provided by the parent class, it's easiest to read if its also here. will update, thanks! 👍